crm-plugin/config/routes/contacttag.json

```diff
@@ -0,0 +1,28 @@
+{
+  "routes": [
+    {
```

Comment:
where are other routes for contacttag. since it extends base table should have all methods of base file exposed in api